util/World.lua

```diff
@@ -0,0 +1,21 @@
+World               = {}
+local metaWorld     = {
+    __newindex      = function(t, k, v)
+        k = nil
+        v = nil
+        return
+    end,
+    __metatable     = false
+}
+
+ceres.addHook("main::before", function()
+    World.RECT      = GetWorldBounds()
+    World.REG       = CreateRegion()
+    World.MAX_X     = GetRectMaxX(World.RECT)
+    World.MAX_Y     = GetRectMaxY(World.RECT)
+    World.MIN_X     = GetRectMinX(World.RECT)
+    World.MIN_Y     = GetRectMinY(World.RECT)
+    setmetatable(World, metaWorld)

```

Comment:
Also, if this metatable suits only the purpose of not modifying the object, I suggest moving it as a unique module into .base scope, so it can be reused for other const objects.
Could be named `base.Const` or something like that.